src/components/FormTodo/FormTodo.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class FormTodo extends React.Component {
+  state = {
+    title: '',
+    id: 1,
+  };
+
+  handleClick = (event) => {
+    event.preventDefault();
+    if (event.target.text.value.length === 0) {
+      return false;
+    }
+
+    const { addTodo } = this.props;
+    const { title, id } = this.state;
+    const newTodo = {
+      title,
+      completed: false,
+      id,
+    };
+
+    this.setState(prevState => ({ title: '', id: prevState.id + 1 }));
+
+    addTodo(newTodo);
+  };
+
+  handleChange = ({ target }) => {
+    this.setState({ title: target.value });
```

Comment:
следует очищать пробелы по бокам
```suggestion
    this.setState({ title: target.value.trim() });
```